### ISSUE TITLE
Switch Between Providers with Command Palette and Hotkeys

### ISSUE THREAD

**Issue by Bluemoondragon07** [cloudy9101/obsidian-image-inserter#12]
For some types of images, like art or decoration, it's better to browse Pixabay. For other images, such as realistic references, it's better to browse Unsplash. I switch between the two depending on the kind of image I need. 
I think a way to switch between the two providers using the command palette or a hotkey would be a really neat addition.
Also, thank you for creating and maintaining this amazing plug-in.

**Comment by cloudy9101** [comment]
Hi @Bluemoondragon07 
Definitely a great idea. I released a new version ([0.4.0](https://github.com/cloudy9101/obsidian-image-inserter/releases/tag/0.4.0)) which supported select image providers when searching images in the modal. The selector is just on the right of the search input. In the image searching modal, I also added a shortcut key binding for changing the image provider which is `ctrl + u`.
Give it a try and hope you will enjoy this addition.
